operations/network/management_network/snmp_exporter_configs.md

```diff
@@ -0,0 +1,40 @@
+## Prometheus SNMP Exporter Deployment
+rometheus SNMP Exporter is deployed with the `cray-sysmgmt-health` chart to the `sysmgmt-health` namespace as part of the Cray System Management \(CSM\) release.
+
+### Customizations
+
+For a complete set of available settings, consult the values.yaml file for the `cray-sysmgmt-health` chart. The most common customizations to set are specified in the following table. They must be set in the customizations.yaml file under the `prometheus-snmp-exporter:` setting.
+
+|Customization|Default|Description|
+|-------------|-------|-----------|
+|`serviceMonitor.enabled`|`true`|Enables serviceMonitor for snmp exporter \(default chart value is `true`\)|
+|`params.enabled`|`false`|Sets the snmp exporter params change to true \(default chart value is `false`\)|
+|`params.conf.module`|`if_mib`| snmp exporter to select which module \(default chart value is `if_mib`\)|
+|`params.conf.target`|`127.0.0.1`| add list of switch targets to snmp exporter to monitor \(default chart value is `127.0.0.1`\)|
+
+prometheus-snmp-exporter block will look like below in customizations.yaml
+
+      cray-sysmgmt-health:
+	    prometheus-snmp-exporter:
+          serviceMonitor:
+            enabled: true
+            params:
+              enabled: true
+              conf:
+                module:
+                - if_mib
+                target:
+                - 127.0.0.1
+
+#### Obtain the list of switches to as targets
```

Comment:
Looks like a word is missing. Change to:

Obtain the list of switches to use as targets